client/src/styles/Theme.js

```diff
@@ -1,6 +1,6 @@
 export const theme = {
-    primaryDark: '#0D0C1D',
-    primaryLight: '#EFFFFA',
-    primaryHover: '#343078',
-    mobile: '576px',
-}
\ No newline at end of file
+	primaryDark: '#0D0C1D',
```

Comment:
It looks like you've changed the whitespace from 4 space indentation to 1 tab. 
This doesn't make a difference to the running of the code, but it does make PRs a bit more confusing and harder to see what has actually changed. You should be able to set up a linter or your IDE to use the same settings for the project for your whole team.

Be warned, there have been many wars fought over Tabs vs Spaces over the years...